packages/boot/src/mixins/boot.mixin.ts

```diff
@@ -53,6 +55,11 @@ export function BootMixin<T extends Constructor<any>>(superClass: T) {
       this.bind(BootBindings.BOOT_OPTIONS).toDynamicValue(
         () => this.bootOptions,
       );
+
+      this.bind(CoreBindings.APPLICATION_PACKAGE_JSON).toDynamicValue(() => {
```

Comment:
Would it not be better for this to be done via a `Booter`? This is not the intended usage of `bootmixin` and seems like concerns are being mixed.

Reply:
I don't mind moving it to a `application` booter in the future once we figure out our configuration loading story, such as how to deal with env vars.